src/Constants.h

```diff
@@ -10,6 +10,30 @@ constexpr size_t PACKET_PAYLOAD_SIZE = 8;
 constexpr double SHOULDER_LENGTH = 0.6; // placeholder(m)
 constexpr double ELBOW_LENGTH = 0.7;	// placeholder(m)
 
+/**
+   Number of millidegrees per degree
+ */
+constexpr float MDEG_PER_DEG = 1000;
+
+constexpr std::array<uint32_t, 6> arm_PPJRs = {
+	17 * 1000, // base, estimate
+
+	20 * 1000, // shoulder, estimate
+	36 * 1000, // elbow, rough estimate
+
+	360 * 1000, // forearm, unmeasured
+	360 * 1000, // diff_left, unmeasured
+	360 * 1000	// diff_right, unmeasured
+};
+
+// So far only the base, shoulder, elbow have been tuned
+//
+// base, shoulder, elbow, forearm, diff_left, diff_right
+constexpr std::array<int32_t, 6> arm_Ps = {1000, 100, 500, 0, 0, 0};
+constexpr std::array<int32_t, 6> arm_Is = {50, 0, 50, 0, 0, 0};
+constexpr std::array<int32_t, 6> arm_Ds = {10000, 1000, 10000, 0, 0, 0};
+constexpr std::array<uint8_t, 6> arm_encoder_signs = {0, 0, 1, 0, 0, 0};
+
```

Comment:
These look like physical constants, should these be in `real_world_constants.h` instead? The simulator probably wouldn't have the same values/constants.

Reply:
Yeah, not sure why I put those there - I think since I cherry-picked these commits from the protocol branch there's another commit that when composed with this one will move them to `real_world_constants.h`. I'll remove this.